g3w-admin/qdjango/sitetrees.py

```diff
@@ -2,67 +2,164 @@
-           in_menu=False, url_as_pattern=True, icon_css_class='fa fa-edit'),
-      item('Update QGIS project\'s message {{ group.slug }} {{ project.slug }}',
-           'qdjango-project-messages-update group.slug project.slug object.pk',
-           in_menu=False, url_as_pattern=True, icon_css_class='fa fa-edit')
-  ]),
+    # ITALIAN
+    G3Wtree(
+          'qdjango',
+          title='Qdjango',
+          module='qdjango',
+          items=[
+               item(
+                    'Aggiungi progetto QGIS {{ group.slug }}',
+                    'qdjango-project-add group.slug',
+                    in_menu=False,
+                    url_as_pattern=True,
+                    icon_css_class='fa fa-plus',
+                    access_by_perms=['qdjango.add_project']
+               ),
+               item(
+                    'Agg. progetto QGIS {{ group.slug }} {{ object.slug }}',
+                    'qdjango-project-update group.slug object.slug',
+                    in_menu=False,
+                    url_as_pattern=True,
+                    icon_css_class='fa fa-edit'
+               ),
+               item(
+                    'Lista strati progetto QGIS {{ group.slug }} {{ object.slug }}',
+                    'qdjango-project-layers-list group.slug project_slug',
+                    in_menu=False,
+                    url_as_pattern=True,
+                    icon_css_class='fa fa-edit'
+               ),
+               item(
+                    'Lista messaggi progetto QGIS {{ group.slug }} {{ project.slug }}',
+                    'qdjango-project-messages-list group.slug project.slug',
+                    in_menu=False,
+                    url_as_pattern=True,
+                    icon_css_class='fa fa-edit'
+               ),
+               item(
+                    'Aggiungi messaggio progetto QGIS {{ group.slug }} {{ project.slug }}',
+                    'qdjango-project-messages-add group.slug project.slug',
+                    in_menu=False,
+                    url_as_pattern=True,
+                    icon_css_class='fa fa-edit'
+               ),
+               item(
+                    'Agg. messaggio progetto QGIS {{ group.slug }} {{ project.slug }}',
+                    'qdjango-project-messages-update group.slug project.slug object.pk',
+                    in_menu=False,
+                    url_as_pattern=True,
+                    icon_css_class='fa fa-edit'
+               )
+          ]
+     ),
+
+     # ENGLISH
+     G3Wtree(
+          'qdjango_en',
+          title='Qdjango',
+          module='qdjango',
+          items=[
+               item(
+                    'Add QGIS project {{ group.slug }}',
+                    'qdjango-project-add group.slug',
+                    url_as_pattern=True,
+                    icon_css_class='fa fa-plus',
+                    in_menu=False,
+                    access_by_perms=['qdjango.add_project']
+               ),
```

Comment:
@Raruto  no because as you can read on officila django-sitetree documentation: https://django-sitetree.readthedocs.io/en/latest/i18n.html, you have to create `my_tree_{locale-id}` alias.

> Given that my_tree_en, my_tree_ru and other my_tree_{locale-id}-like trees are considered internationalization sitetrees. These are used (if available) in accordance with current locale used in project.